openbadges/.htaccess

```diff
@@ -1,4 +1,5 @@
 Options +FollowSymLinks
 RewriteEngine on
 RewriteRule ^v1$ https://dev.truecred.com/contexts/openbadges-v1.jsonld [R=302,L]
+RewriteRule ^v1.1$ http://specification.openbadges.org/1.1/context.json [R=302,L]
```

Comment:
We need to talk about this line on the call tomorrow @davidlehn @dlongley @ottonomy. Doing a v1.1 context, while the path of least resistance, doesn't make us have the important discussion wrt. how we're going to merge the Open Credentials and Open Badges vocabularies. I don't have an opinion on how the merge should happen... just that it /should/ happen and we're hitting the point where it needs to happen.